src/js/player.js

```diff
@@ -425,6 +425,15 @@ class Player extends Component {
       this.autoplay(this.options_.autoplay);
     }
 
+    // check plugins
+    if (options.plugins) {
```

Comment:
If we throw an error after making the player `evented` it causes a dispose handler to be created and then lost due to the error. We can either do this, or trigger dispose on the player below before we throw the error but I think this made more sense.